src/main/java/ch/vorburger/fswatch/DirectoryWatcherImpl.java

```diff
@@ -52,12 +55,27 @@
 
     protected final WatchService watcher = FileSystems.getDefault().newWatchService(); // better final, as it will be accessed by both threads (normally OK either way, but still)
     protected final Thread thread;
-
-    // protected because typical code should use the DirectoryWatcherBuilder instead of this directly
+    protected final List<ChangeKind> changeKindsList = new ArrayList<>();
+    
+    /*
+     * Retained the existing constructor and it will retain the old behaviour of responding only to MODIFIED and DELETED ChangeKinds
+     */
     protected DirectoryWatcherImpl(boolean watchSubDirectories, final Path watchBasePath, final Listener listener, FileFilter fileFilter, ExceptionHandler exceptionHandler) throws IOException {
+    	this(watchSubDirectories,watchBasePath,listener,fileFilter,exceptionHandler, new ChangeKind[] {ChangeKind.MODIFIED, ChangeKind.DELETED});
+    }
+    
+    	   
+
+    /*protected because typical code should use the DirectoryWatcherBuilder instead of this directly
+    /*
+     * Added ChangeKinds[] as an arguments to enable clients to specify what ChangeKinds that they are interested in. For e.g on mac
+     * I still want to get CREATED events as mac's behaviour seems to be diff from linux.
+     */
```

Comment:
this is interesting, and would perhaps be worthwhile to mention more prominently directly on the README? (FYI I don't have a Mac, only Linux)